tests/unit/backend/utils/field-validation/email-validation.spec.ts

```diff
@@ -0,0 +1,149 @@
+import EmailValidator = require('../../../../../src/app/utils/field-validation/validators/EmailValidator.class')
```

Comment:
Use `import <x> from '<z>'` syntax instead :D
Tests have [baseUrl of root](https://www.typescriptlang.org/docs/handbook/module-resolution.html#base-url) enabled, so it can be shortened to

```ts
import EmailValidator from 'src/app/utils/field-validation/validators/EmailValidator.class'
import validateField from 'src/app/utils/field-validation'
```

Reply:
Nice. Thanks